obspy/core/event/base.py

```diff
@@ -1166,16 +1166,9 @@ class DataUsed(__DataUsed):
     moment-tensor inversion.
 
     :type wave_type: str or :class:`~obspy.core.event.header.DataUsedWaveType`
```

Comment:
This is the type though, not the description, which already says that. Though sure, you can't actually create anything with a `DataUsedWaveType` type...